src/screens/FeedScreen/RenderList.js

```diff
@@ -213,7 +228,8 @@ const RenderListFeed = (props) => {
             statusVote={voteStatus}
             showScoreButton={showScoreButton}
             onPressScore={() => showScoreAlertDialog(item)}
-            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
+            isSelf={isSelf}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isSelf` seems to have been changed from a comparison operation to a direct value. Ensure that this change doesn't affect the logic of the component.

```diff
-            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
+            isSelf={isSelf}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->